extensions/rich_text_components/Video/directives/video.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="oppia-noninteractive-video-iframe-container">
+  <div class="oppia-noninteractive-video-iframe">
+    <youtube-player class="protractor-youtube-player"
+                    [videoId]="videoId"
+                    [startSeconds]="start"
+                    [endSeconds]="end"
+                    [playerVars]="playerVars">
+    </youtube-player>
+  </div>
+</div>
+<style>
+  /* The value of 60.9% is derived from the ratio of the height to the width
+   * (280/480). See http://alistapart.com/article/creating-intrinsic-ratios-for-video
+   * for more details.
+   */
+  .oppia-noninteractive-video-iframe-container {
+    padding-bottom: 60.9%;
+    position: relative;
+  }
+
+  .oppia-noninteractive-video-iframe /deep/ iframe {
```

Comment:
Mistake? (`/deep/`)

Reply:
We need deep.

Reply:
Hmm, can we add a comment for this, this seems like quite an uncommon selector that is also quite hard to find and it seems that there are plans to deprecate it.

Reply:
https://github.com/oppia/oppia/issues/12898 Created an issue to track (Also this can be very easily tracked in the codebase by searching for `/deep/`). The fix for this is simple. We are using it in quite some places at the moment though.